### PR TITLE
SNMP: Add plugin manifest

### PR DESCRIPTION
**Pull request by cakekoa**
# What does this PR do?

Fixes part of #3234.

- Adds a plugin manifest for the SNMP plugin

## PR Checklist
* [x] Have you added an explanation of what your changes do and why you'd like to include them?
* [ ] Is the TravisCI build passing?
* [ ] Was the CHANGELOG.md updated to reflect the changes?
* [ ] Was the documentation framework updated to reflect the changes?
* [ ] Have you checked that you haven't introduced any duplicate code?

## Testing Checklist

* [ ] Added relevant unit tests?
* [x] Do all unit tests pass?
* [ ] Do all end-to-end tests pass?
* [ ] Any other testing performed?
    > Tested by {Running the Monkey locally with relevant config/running Island/...}
* [ ] If applicable, add screenshots or log transcripts of the feature working

### CODE REVIEW

File: monkey/agent_plugins/exploiters/snmp/manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+remediation_suggestion: >-
+    Configure SNMP to use read-only communities.
+
+    Limit access over SNMP to trusted hosts.
+
+    Use SNMPv3 with the authPriv security level if possible.
+
+
+    The machine is vulnerable to an SNMP attack.
+    An Infection Monkey Agent executed a command over SNMP using stolen/configured credentials.
```
Comment (mssalvatore) [inline monkey/agent_plugins/exploiters/snmp/manifest.yaml:21]: Should we be more specific in here, call out the CVE, and recommend they Install a patched version?

File: monkey/agent_plugins/exploiters/snmp/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+name: SNMP
+plugin_type: Exploiter
+supported_operating_systems:
+  - linux
+  - windows
+target_operating_systems:
+  - linux
+title: SNMP Exploiter
+version: 1.0.0
+description: Attempts remote command execution over SNMP using known credentials.
```
Comment (shreyamalviya) [inline monkey/agent_plugins/exploiters/snmp/manifest.yaml:10]: We should mention brute-force.

File: monkey/agent_plugins/exploiters/snmp/manifest.yaml (hunk 1 above)
Comment (shreyamalviya) [reply monkey/agent_plugins/exploiters/snmp/manifest.yaml:21]: +1
We can do it the way we've done it for Zerologon.
We should also do this for Hadoop.

Comment (cakekoa) [reply monkey/agent_plugins/exploiters/snmp/manifest.yaml:21]: Is CVE-2020-15862 the correct one?

Comment (mssalvatore) [reply monkey/agent_plugins/exploiters/snmp/manifest.yaml:21]: Looks good.

Comment (mssalvatore) [reply monkey/agent_plugins/exploiters/snmp/manifest.yaml:21]: > We should also do this for Hadoop.

@shreyamalviya  I don't believe there's any specific CVE associated with Hadoop.

**Pull request merged**